exercises/go-counting/canonical-data.json

```diff
@@ -0,0 +1,172 @@
+{
+  "exercise": "go-counting",
+  "version": "1.0.0",
+  "cases": [
+    {
+      "description": "Test for black territory on 5x5 board",
+      "property": "territory",
+      "input": {
+        "board": [
+                    "  B  ",
+                    " B B ",
+                    "B W B",
+                    " W W ",
+                    "  W  "
+                 ],
+        "x": 0,
+        "y": 1
+      },
+      "expected": {
+        "stone": "BLACK",
+        "territory": [[0, 0], [0, 1], [1, 0]]
+      }
+    },
+    {
+      "description": "Test for white territory on 5x5 board",
+      "property": "territory",
+      "input": {
+        "board": [
+                    "  B  ",
+                    " B B ",
+                    "B W B",
+                    " W W ",
+                    "  W  "
+                ],
+        "x": 2,
+        "y": 3
+      },
+      "expected": {
+        "stone": "WHITE",
+        "territory": [[2, 3]]
+      }
+    },
+    {
+      "description": "Test for open territory on 5x5 board",
+      "property": "territory",
+      "input": {
+        "board": [
+                    "  B  ",
+                    " B B ",
+                    "B W B",
+                    " W W ",
+                    "  W  "
+                ],
+        "x": 1,
+        "y": 4
+      },
+      "expected": {
+        "stone": "NONE",
+        "territory": [[0, 3], [0, 4], [1, 4]]
+      }
+    },
+    {
+      "description": "Test for none territory on 5x5 board",
+      "property": "territory",
+      "input": {
+        "board": [
+                    "  B  ",
+                    " B B ",
+                    "B W B",
+                    " W W ",
+                    "  W  "
+                ],
+        "x": 1,
+        "y": 1
+      },
+      "expected": {
+        "stone": "NONE",
+        "territory": []
```

Comment:
I'm wondering whether noting a stone as `"stone": "NONE"` and `"territory": []`  is sufficiently distinctive from a territory with no owner. I was considering whether this should be considered an `error` instead. But I do see that `"territory": []` is distinguishable because anything that's not a stone will have at least one element in here.

So, would anyone like to argue either for or against having this be an `error`?